### PR TITLE
[feat]: [PL-25264]: Removing Duplicate Redis entries

### PR DESCRIPTION
**Pull request by mankrit-singh**
Please read [go/trigger-cmd](https://harness.atlassian.net/wiki/spaces/BT/pages/21082441647/PR+Checks) to re-trigger PR Checks using Github comments.

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/33470)

### CODE REVIEW

**Comment by mankrit-singh** [comment]
trigger messagecheck

**Review by archit-harness — APPROVED** [review]
> Reviewed 1 of 1 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33470)* status: 2 file(s) to review. 1 of 2 approvals obtained.

File: 955-setup-usage-sdk/src/main/java/io/harness/ng/core/entitysetupusage/entity/EntitySetupUsage.java [hunk 1]
```diff
@@ -145,4 +146,16 @@ public static final class EntitySetupUsageKeys {
...
+  @Override
+  public int hashCode() {
+    return Objects.hash(referredByEntityType, referredByEntityFQN, referredByEntityRepoIdentifier,
+        referredByEntityBranch, referredEntityType, referredEntityFQN, referredEntityRepoIdentifier,
+        referredEntityBranch, accountIdentifier);
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    return this.hashCode() == o.hashCode();
```
Comment (ashishsanodia-harness) [inline 955-setup-usage-sdk/src/main/java/io/harness/ng/core/entitysetupusage/entity/EntitySetupUsage.java:159]: Why you are checking just for hashcode equality? You can use @EqualsAndHashCode from lombok

**Comment by mankrit-singh** [comment]
trigger authorcheck

**Review by DeepakPatankar — COMMENTED** [review]
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33470)* status: 2 file(s) to review. 1 change requests. 0 of 2 approvals obtained.

**Review by DeepakPatankar — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33470)* status: 2 file(s) to review. 1 of 2 approvals obtained.

**Review by ashishsanodia-harness — APPROVED** [review]
> Reviewed 1 of 1 files at r3, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33470)* status: :shipit: complete! ✓

**Pull request merged**